.github/workflows/ci-cd.yml

```
@@ -387,8 +387,14 @@ jobs:
         - pre-commit
         - metadata-validation
         environment-variables:
+        # `no-commit-to-branch` is skipped because it does not make sense
+        # in the CI, only locally.
+        # Ref: https://github.com/pre-commit/pre-commit-hooks/issues/1124
         - >-  # only affects pre-commit, set for all for simplicity:
-          SKIP=hadolint,shfmt
+          SKIP=
+          hadolint,
+          no-commit-to-branch,
+          shfmt,
```

just curios, can we have this?
```suggestion
          SKIP=
            hadolint,
            no-commit-to-branch,
            shfmt,
```

IMO, that will make it more readable, as it specify visually that skipped values are actually values for SKIP key

---

Sometimes, we can, sometimes we can't. I think I've seen some corner cases when using `>-`, which is why I usually don't add those indents.

But yes, you're right, it's more readable. So let's try it and see whether it breaks.